pkg/instance/manager.go

```diff
@@ -8,7 +8,11 @@ import (
 	"strings"
 	"time"
 
-	"github.com/cozy/cozy-stack/pkg/utils"
+	"github.com/cozy/cozy-stack/pkg/config"
```

Comment:
Do you use `gofmt` or `goimports`? It is weird to have a blank line here

Reply:
I use goimports

Reply:
It's weird, goimports remove the blank lines on my machine.